fe/estimator_abfe.py

```diff
@@ -24,17 +24,16 @@
 class SimulationResult:
     xs: np.array
     boxes: np.array
-    du_dps: np.array
     lambda_us: np.array
 
 
 def flatten(v):
-    return tuple(), (v.xs, v.boxes, v.du_dps, v.lambda_us)
+    return tuple(), (v.xs, v.boxes, v.lambda_us)
 
 
 def unflatten(aux_data, children):
-    xs, boxes, du_dps, lambda_us = aux_data
-    return SimulationResult(xs, boxes, du_dps, lambda_us)
+    xs, boxes, lambda_us = aux_data
+    return SimulationResult(xs, boxes, lambda_us)
 
 
 jax.tree_util.register_pytree_node(SimulationResult, flatten, unflatten)
```

Comment:
Ditto, probably unused since `grad(delta_G)` is deprecated / removed

Reply:
Removed in 3ab3b17